src/main/target/OMNIBUSF4FW/target.c

```diff
@@ -32,22 +32,33 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR,           0, 0), // D(1,7) U(1,2)
     DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MOTOR,           0, 0), // D(1,2) U(1,2)
     DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MOTOR,           0, 1), // D(1,6) U(1,7)
-#if defined(OMNIBUSF4FW)
+#if defined(OMNIBUSF4FW) || defined(OMNIBUSF4V6)
```

Comment:
This won't do what you think it does - `OMNIBUSF4FW` is always defined, since it is the base target.